app/src/main/java/org/gem/indo/dooit/views/helpers/activity/CurrencyHelper.java

```diff
@@ -9,9 +12,17 @@
  */
 
 public class CurrencyHelper {
-
+    public static double DEFAULT_VALUE = 0.0f;
     public static String format(Object o) {
-        return NumberFormat.getCurrencyInstance().format(Float.parseFloat(String.valueOf(o)));
+        try {
+            String value = String.valueOf(o);
+            BigDecimal number = new BigDecimal(value);
+            return NumberFormat.getCurrencyInstance().format(number);
```

Comment:
Why use BigDecimal and not 
` return NumberFormat.getCurrencyInstance().format(Double.format(String.valueOf(o)))` ?